src/components/Navbar.js

```diff
@@ -1,16 +1,22 @@
 import { Link } from 'react-router-dom';
+import { BiSolidGroup } from 'react-icons/bi';
 
 const Books = () => (
-  <div className="navbar">
-    <h1>Bookstore CMS</h1>
-    <ul className="">
-      <li className="">
-        <Link to="/">Books</Link>
-      </li>
-      <li>
-        <Link to="/categories">Categories</Link>
-      </li>
-    </ul>
+  <div className="navbarP">
+    <div className="navbar">
+      <h1>Bookstore CMS</h1>
```

Comment:
- [ ] You're doing a great job with your project, but it's crucial to ensure that your styling aligns precisely with the provided design on Zeplin, as per the project requirements. Currently, the logo doesn't adhere to the design's specifications in terms of font weight, line height, font size, color, and other style attributes. To achieve the desired styling for your logo, I recommend closely following the CSS rules outlined in Zeplin. By doing so, you'll ensure that your website's logo matches the design as closely as possible, creating a visually consistent and polished appearance.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/BilalLiaquat7/bookstore/assets/55840999/dc1d39b6-5aa6-4b69-b6a8-58d9addff30f)| ![image](https://github.com/BilalLiaquat7/bookstore/assets/55840999/8dbd1083-a6bb-4023-8238-8e45bbd4fc6b)|

Maintaining strict adherence to the design guidelines is essential for achieving a professional and cohesive look for your project. Keep up the great work, and thank you for your attention to detail! 🌟🎨